### DISCUSSION TITLE
2024년을 보내고 2025년을 맞이하며 - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#66] [Announcements]
# 2024년을 보내고 2025년을 맞이하며 - 정현수 기술 블로그

2024년 회고

https://junghyeonsu.com/posts/2024-retrospect/

**Comment by irrationnelle** [comment]
풍성하니 꽉 찬 24년도 였군요. 25년도 그러할 겁니다. 응원합니다!
